<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogData</class>
 <widget class="QDialog" name="DialogData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBoxData">
     <property name="title">
      <string>Data</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QComboBox" name="comboBoxGenotype">
        <item>
         <property name="text">
          <string>VCF</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PED/MAP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>HapMap</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Genotype</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditGenotype"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButtonGenotype">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxOptions">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditMinMAF"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditMaxHeteroz"/>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pushButtonIndividual">
        <property name="text">
         <string>Indiv list file...</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEditMaxLocusMissing"/>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="labelMinAllele">
        <property name="text">
         <string>Min. alleles:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelMaxIndivMissing">
        <property name="text">
         <string>Max. per-indiv missing rate:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEditMaxIndivMissing"/>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="labelMaxAllele">
        <property name="text">
         <string>Max. alleles:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLineEdit" name="lineEditMaxAllele"/>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="lineEditLocus"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelOutputFormat">
        <property name="text">
         <string>Output format:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelMaxLocusMissing">
        <property name="text">
         <string>Max. per-locus missing rate:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="lineEditChromosome"/>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxOutputFormat">
        <item>
         <property name="text">
          <string>VCF</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PED/MAP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>HapMap</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Genotype</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButtonLocus">
        <property name="text">
         <string>Loci list file...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="lineEditIndividual"/>
      </item>
      <item row="3" column="3">
       <widget class="QLineEdit" name="lineEditMinAllele"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelMaxHeteroz">
        <property name="text">
         <string>Max. heterozygosity rate:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labelChromosome">
        <property name="text">
         <string>Chromosome:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelMinMAF">
        <property name="text">
         <string>Min. minor allele frequency:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="checkBoxDiploid">
        <property name="text">
         <string>Force diploid genotype</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBoxGenotype</tabstop>
  <tabstop>lineEditGenotype</tabstop>
  <tabstop>pushButtonGenotype</tabstop>
  <tabstop>comboBoxOutputFormat</tabstop>
  <tabstop>lineEditMinMAF</tabstop>
  <tabstop>lineEditMaxHeteroz</tabstop>
  <tabstop>lineEditMaxLocusMissing</tabstop>
  <tabstop>lineEditMaxIndivMissing</tabstop>
  <tabstop>checkBoxDiploid</tabstop>
  <tabstop>lineEditChromosome</tabstop>
  <tabstop>pushButtonLocus</tabstop>
  <tabstop>lineEditLocus</tabstop>
  <tabstop>pushButtonIndividual</tabstop>
  <tabstop>lineEditIndividual</tabstop>
  <tabstop>lineEditMinAllele</tabstop>
  <tabstop>lineEditMaxAllele</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogData</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogData</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
